py-polars/tests/unit/io/test_csv.py

```diff
@@ -1266,6 +1266,20 @@ def test_csv_categorical_categorical_merge() -> None:
     )["x"].to_list() == ["A", "B"]
 
 
+def test_csv_clip_n_rows_to_sample_size() -> None:
```

Comment:
The `tmp_path` fixture should be used here - you can figure out how it works from the other I/O tests:

```suggestion
@pytest.mark.write_disk()
def test_csv_clip_n_rows_to_sample_size(tmp_path: Path) -> None:
```

Reply:
Thanks for the suggestion, just updated!